webpack.config.js

```diff
@@ -1,22 +1,27 @@
 const nodeExternals = require('webpack-node-externals')
+const path = require('path')
 
 module.exports = {
-  entry: './app/index.js',
-  target: 'electron',
-  externals: [nodeExternals()],
+  entry: path.join('.', 'app', 'index.js'),
```

Comment:
for consistency can we replace `.` by `__dirname`? 

Also, because we know the path we can write `path.normalize('./app/index.js')` so that it works on windows (same below 
```path.normalize(`${__dirname}/static`)```) [`path.join` calls `normalize`](https://github.com/nodejs/node/blob/master/lib/path.js#L490).

Reply:
Oh, just read the code and it also uses normalize on mac. Learning new things every day. (thx)